### ISSUE TITLE
Keyboard navigation doesn't work on Menubar

### ISSUE THREAD

**Issue by meanstackmax** [primefaces/primereact#7898]
### Describe the bug

**Describe the bug**
I've updated Primereact to the latest version (10.9.4) and noticed that keyboard navigation doesn't work anymore for the Menubar. Adding an `onKeydown` event handler to the menuitem with pass through options will solve the issue just partially. While this work for the 0 level menu items it still doesn't work for the submenu items. I've also noticed that `onKeydown` handler isn't being passed to the nested items.

**Additional info**
I was using both: custom templates and without any template for the menuItem.

### Reproducer

https://stackblitz.com/edit/e8tdia?file=src%2FApp.jsx,package.json

### System Information

```Shell
`primereact 10.9.4`
```

### Steps to reproduce the behavior

1. Open stackblitz.
2. Press `tab` to focus the first element. 
3. Notice that the element isn't being focused.

### Expected behavior

Pressing `tab` should focus the first element and the further navigation with arrows should be enabled.

**Comment by fr3ddy** [comment]
Is there already an estimate, when this bug will be worked on? Thank you
